src/Session.js

```diff
@@ -7,9 +9,13 @@ function Session(server, socket) {
     var self = this;
 
     this._socket = socket;
+
+	this._userLocked = false;
```

Comment:
looks like your spacing / tabs might be different than the original project. I'd suggest conforming to what the project is already using.